### PR TITLE
Axis visible in 3D

### PR DESCRIPTION
**Pull request by etpinard**
to be merged in https://github.com/plotly/plotly.js/pull/1595

This PR implements `scene.(x|y|z)axis.visible` at the convert step.

### CODE REVIEW

**Comment by etpinard** [comment]
cc @rreusser

File: src/plots/gl3d/layout/convert.js [hunk 1]
```diff
@@ -64,9 +62,9 @@ function AxesOptions() {
                               [0.8, 0.8, 0.8, 0.5] ];
 
     // some default values are stored for applying model transforms
-    this._defaultTickPad = arrayCopy1D(this.tickPad);
-    this._defaultLabelPad = arrayCopy1D(this.labelPad);
-    this._defaultLineTickLength = arrayCopy1D(this.lineTickLength);
+    this._defaultTickPad = Lib.extendFlat([], this.tickPad);
+    this._defaultLabelPad = Lib.extendFlat([], this.labelPad);
+    this._defaultLineTickLength = Lib.extendFlat([], this.lineTickLength);
```
Comment (alexcjohnson) [inline src/plots/gl3d/layout/convert.js]: isn't this just `.slice()`?

**Comment by alexcjohnson** [comment]
Looks great to me - I can see this (in both 2D and 3D) being pretty nice for certain abstract plots!

💃  after my [one comment](https://github.com/plotly/plotly.js/pull/1599#pullrequestreview-32780770).

File: src/plots/gl3d/layout/convert.js (hunk 1 above)
Comment (etpinard) [inline src/plots/gl3d/layout/convert.js]: Oh right. Good :eye: thanks!

Comment (etpinard) [inline src/plots/gl3d/layout/convert.js]: done in https://github.com/plotly/plotly.js/pull/1599/commits/bc0f8902f00ad2e29e59140a2cebe23db4ed6eea

**Comment by rreusser** [comment]
Definitely! Nice solution!

**Pull request merged**